addons/sale_stock/models/stock.py

```diff
@@ -65,7 +65,7 @@ def _action_synch_order(self):
                 so_line_vals['price_unit'] = 0
             # New lines should be added at the bottom of the SO (higher sequence number)
             if not so_line:
```

Comment:
```suggestion
            if not so_line and sale_order.order_line:
```
Add sequence when order_line avaliable and sale_order